ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.scss

```diff
@@ -16,8 +16,8 @@
     }
 
     &--inverted {
-        background-color: black;
-        color: white;
+        background-color: #303237 !important;
+        color: #fff !important;
```

Comment:
Curious, which browser are you using?  I applied this change but I'm not seeing the correct behaviour.  Also, I'm wondering about the previous change that was made to this file, which seems to have introduced this problem.

Reply:
Chrome, it's working fine on my side the only thing I wanted to add more was on switching to dark mode in appearance for whole UI the toggle button should toggle on its own and revert to light mode on Switch. and yes themify preferences introduced earlier might be overriding these specific tstyles.

Reply:
Thanks for the quick response.  Is that the reason why you created https://github.com/argoproj/argo-cd/pull/17241?  (There wasn't much detail about the problem definition)

To clarify and help me understand correctly the issue, then without the pod log viewer changes from 17241, these are the steps to reproduce the problem:

1. Ensure that in the Settings, Dark Theme is disabled (so background is white)
2. In the pod logs view, the Dark Mode button is not pressed (so the pod viewer background in white).  
3. Go to the Settings page and enable Dark Theme
4. Go back to the pod logs view 
---> ISSUE: The pod logs view is NOT in dark mode, and the background is  _still_  white, but you want this to be in Dark mode immediately upon visiting this page.
--> Similarly, if it was reversed, if the Dark Theme is enabled at first, then you are expecting that when revisiting the pod logs view, Dark Mode should be enabled and the background should be black, regardless of what that setting was?

I think this is a fair complaint.  But one can argue that 1) it's as designed because the Dark mode button should override the 'global' setting from the Settings page. (Likely, the pod viewer feature was merged before the dark theme support went in). 2) Perhaps some users prefer the light theme overall, and want the log viewer to be in dark mode?

So, I think the best thing to do now is to undo 17241. This will fix 18405.

For the issue I think you are describing for 17241, I don't believe it is a css change.